src/include/rbd/librbd.hpp

```diff
@@ -15,14 +15,18 @@
 #ifndef __LIBRBD_HPP
 #define __LIBRBD_HPP
 
+#include <boost/variant.hpp>
 #include <stdbool.h>
 #include <string>
 #include <list>
 #include <map>
 #include <vector>
+#include "common/Formatter.h"
```

Comment:
This is not part of the public API -- neither are the other two new includes, so you cannot add them.